shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/consistencycheck/ConsistencyCheckJob.java

```diff
@@ -74,7 +74,7 @@ public void execute(final ShardingContext shardingContext) {
         }
         Map<String, String> checkResultMap = new LinkedHashMap<>();
         for (Entry<String, DataConsistencyCheckResult> entry : dataConsistencyCheckResult.entrySet()) {
-            YamlDataConsistencyCheckResultValue checkResult = CHECK_RESULT_SWAPPER.swapToYamlConfiguration(entry.getValue());
+            YamlDataConsistencyCheckResult checkResult = CHECK_RESULT_SWAPPER.swapToYamlConfiguration(entry.getValue());
             checkResultMap.put(entry.getKey(), YamlEngine.marshal(checkResult));
         }
         PipelineAPIFactory.getGovernanceRepositoryAPI().persistCheckJobResult(referredJobId, checkJobId, YamlEngine.marshal(checkResultMap));
```

Comment:
Could we put marshal and unmarshal to the same class, e.g. GovernanceRepositoryAPIImpl